src/coreclr/tools/Common/CommandLineHelpers.cs

```diff
@@ -96,6 +96,7 @@ public static TargetArchitecture GetTargetArchitecture(string token)
                     Architecture.Arm => TargetArchitecture.ARM,
                     Architecture.Arm64 => TargetArchitecture.ARM64,
                     Architecture.LoongArch64 => TargetArchitecture.LoongArch64,
+                    (Architecture)9 => TargetArchitecture.RiscV64, /* TODO: update with Architecture.RiscV64 */
```

Comment:
Is there a tracking work item for this TODO? It would be for the best to keep hard coded numbers like this one for as short as possible.